src/graphql/admin/mongoose_graphql_metric_admin_query.erl

```diff
@@ -0,0 +1,190 @@
+-module(mongoose_graphql_metric_admin_query).
+-behaviour(mongoose_graphql).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose_logger.hrl").
+
+-import(mongoose_graphql_helper, [make_error/2, format_result/2]).
+
+-type metric_result() :: term().
+
+execute(_Ctx, _Obj, <<"getMetrics">>, Args) ->
+    get_metrics(Args);
+execute(_Ctx, _Obj, <<"getMetricsAsDicts">>, Args) ->
+    get_metrics_as_dicts(Args);
+execute(_Ctx, _Obj, <<"getClusterMetricsAsDicts">>, Args) ->
+    get_cluster_metrics_as_dicts(Args).
+
+-spec get_metrics(mongoose_graphql:args()) ->
```

Comment:
The main issue with this resolver module is that it has its own separate logic that can't be reused outside GraphQL. The general idea is that we should have generic API modules, e.g. `mongoose_metrics_api` (we are using the suffix `api` for such modules), that can be also used by CLI and REST API. Could you think if it's possible to extract such logic to an API module? I know that maybe only calls to exometer would be there... or maybe something else? Just to improve code reuse and to avoid calling exometer directly from the resolver module? We will need to rework (or drop entirely) `mongoose_api_metrics` (I know, the name is unfortunate) as well.

Actually for other categories we did such rework as part of the GraphQL work, but I know this PR is already too big to include the rework of `mongoose_api_metrics` in it.

As an example, see how `mod_roster_api` is used both by `mod_commands` and `mongoose_graphql_roster_...` resolver modules. Ideally it should be similar for metrics.